main/java/BeforeAfter.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import shop.Cart;
+import parser.JsonParser;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class BeforeAfter {
+    Cart newCart = null;
+    JsonParser parser = null;
+    RealItem car = null;
+    VirtualItem disk = null;
+
+    @BeforeEach
+    public Cart createCart() {
```

Comment:
Bad practice to return something with help of Before/After methods. Change them to return void

Reply:
Correction is applied.